packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -479,15 +480,18 @@ Future<void> buildGradleAar({
   @required AndroidBuildInfo androidBuildInfo,
   @required String target,
   @required Directory outputDirectory,
+  String buildNumber,
```

Comment:
maybe make this a `@required` field and remove  `buildNumber ??=...` since `build_aar.dart` does that too. Also, document this argument in  the comment above the `buildGradleAar` definition .